install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -4060,3 +4065,21 @@
 	
 	updateToDbVersion('201809280');
 }
+
+
+if (isFroxlorVersion('0.9.39.5') || isDatabaseVersion('201809280')) {
+
+        showUpdateStep("Adding new fields for NGINX-Reverse-Proxy");
+        Database::query("ALTER TABLE `" . TABLE_PANEL_IPSANDPORTS . "` ADD `proxyto` int(11) NOT NULL DEFAULT '0';");
+        lastStepStatus(0);
+
+        showUpdateStep("Updating settings table");
+        Database::query("INSERT INTO `" . TABLE_PANEL_SETTINGS . "` (`settinggroup`, `varname`, `value`) VALUES ('system', 'apache_use_nrp', '0');");
```

Comment:
When adding new Settings, please use Settings::addNew (as used many times in this file)